e2e-tests/ukef/cypress/e2e/journeys/gef/submit-AIN-deal-with-issued-facilities-scheduled-deal-cancellation-maker-unable-to-view-update-issue-facility-section/submit-AIN-deal-with-issued-facilities-scheduled-deal-cancellation-maker-unable-to-view-update-issue-facility-section.spec.js

```diff
@@ -0,0 +1,113 @@
+    //---------------------------------------------------------------
+    // portal maker submits gef deal for review
+    //---------------------------------------------------------------
+    cy.login(BANK1_MAKER1);
+    gefPages.applicationDetails.visit(deal);
+    submitButton().click();
+    cy.url().should('eq', relative(`/gef/application-details/${dealId}/submit`));
+    cy.keyboardInput(gefPages.applicationSubmission.commentsField(), 'go');
+    submitButton().click();
+
+    //---------------------------------------------------------------
+    // portal checker submits gef deal to ukef
+    //---------------------------------------------------------------
+    cy.login(BANK1_CHECKER1);
+    gefPages.applicationDetails.visit(deal);
+    submitButton().click();
+    gefPages.applicationSubmission.confirmSubmissionCheckbox().check();
+    submitButton().click();
+
+    // expect to land on the /submit-to-ukef page with a success message
+    cy.url().should('include', '/submit-to-ukef');
+
+    //---------------------------------------------------------------
+    // user login to TFM and schedule a cancellation gef deal for tomorrow
+    //----------------------------------------------------------------
+    cy.clearCookie('dtfs-session');
+    cy.clearCookie('_csrf');
+    cy.getCookies().should('be.empty');
+    cy.forceVisit(TFM_URL);
+
+    cy.tfmLogin(PIM_USER_1);
+    cy.forceVisit(`${TFM_URL}/case/${dealId}/deal`);
+    tfmPages.caseDealPage.cancelDealButton().click();
+    cy.keyboardInput(tfmPages.reasonForCancellingPage.reasonForCancellingTextBox(), 'cancel');
+    tfmPartials.continueButton().click();
+    cy.url().should('include', '/bank-request-date');
+    cy.completeDateFormFields({ idPrefix: 'bank-request-date' });
+    tfmPartials.continueButton().click();
+
+    cy.url().should('include', '/effective-from-date');
+    cy.completeDateFormFields({ idPrefix: 'effective-from-date', date: tomorrow.date });
+    tfmPartials.continueButton().click();
+
+    cy.url().should('include', '/check-details');
+    tfmPages.checkDetails.dealDeletionButton().click();
+
+    cy.url().should('include', '/deal');
+    tfmPartials.successBanner().should('exist');
+    tfmPartials
+      .successBanner()
+      .invoke('text')
+      .then((text) => {
+        expect(text.trim()).to.contain('scheduled for cancellation');
+      });
+
+    //-----------------------------------------------------------------------
+    // user login to portal and unable to view update issue facility section
+    //-----------------------------------------------------------------------
+
+    cy.clearCookie('dtfs-session');
+    cy.clearCookie('_csrf');
+    cy.getCookies().should('be.empty');
+    cy.login(BANK1_MAKER1);
+    gefPages.applicationDetails.visit(deal);
+    gefPages.applicationPreview.unissuedFacilitiesHeader().should('not.exist');
```

Comment:
I was thinking it will be good to test issued facilities as well. That update facilities section on portal is hidden even if we have only unissued or issued facilities.